### ISSUE TITLE
Error when running the example in README.md

### ISSUE THREAD

**Issue by Maverobot** [hupe1980/golc#7]
First of all, thank you for open sourcing your project! When running the example code from README.md, I got the following error:

```
2023/12/22 20:44:46 All attempts fail:
#1: error, status code: 400, message: Invalid value for 'tool_choice': 'tool_choice' is only allowed when 'tools' are specified.
exit status 1
```

Not sure if I missed something obvious though...

**Comment by hupe1980** [comment]
You haven't forgotten anything. There is a bug. I will release a fix immediately.

**Comment by hupe1980** [comment]
The bug is fixed in release v0.0.99
